src/app/clusters/operational-state-server/operational-state-server.cpp

```diff
@@ -262,6 +262,13 @@ void Instance::InvokeCommand(HandlerContext & handlerContext)
         HandleCommand<Commands::Stop::DecodableType>(handlerContext,
                                                      [this](HandlerContext & ctx, const auto & req) { HandleStopState(ctx, req); });
         break;
+    default:
+        // Should we check if the command is in the base cluster's number-space and if so respond with an error?
+        // Is this not possible due to the data model knowing what commands are available
```

Comment:
I assume that the interaction model will not allow commands that are not enabled in the zap configuration to reach this point. Therefore we do not need to check for this here. Can anyone confirm this?

Reply:
The IM will not allow commands that are not claimed to be supported to reach this point.  The claimed support could be via ZAP config or CommandHandlerInterface claiming the command is supported, right?

But past that, yes, only commands we somehow claim to support will land here.